packages/fxa-admin-panel/jest.config.js

```diff
@@ -1,5 +1,8 @@
 module.exports = {
   roots: ['<rootDir>/src'],
+  moduleNameMapper: {
+    '^@fxa-components/(.*)$': '<rootDir>/../fxa-components/$1',
```

Comment:
@dannycoates in theory this should be working, but Jest is still yelling at me that it can't find the module. Any ideas?